charts/zenith-proxy/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: zenith-proxy
 description: Zenith Proxy
 type: application
-version: 1.0.3
+version: 1.1.0
```

Comment:
Hm, it's technically a major bump since it will break for users who pin `image.tag`. But promoting this to a major change is too severe. Let me think..

Reply:
@cicdteam (I'm new to helm, so correct me if I'm wrong:) Seems like we'd lose the ability to roll back to a previous deploy, since the `deploy-staging-proxy` job in `zenithdb/zenith` picks up the latest chart with `helm upgrade zenith-proxy zenithdb/zenith-proxy`. Wouldn't it be better to merge this repo into zenithdb/zenith? Another benefit of the merge is that this version number would have more meaning. It would be semvering the service itself, not just the deployment strategy.